tests/unit/Dns/Cname/CnameRecordTest.php

```diff
@@ -10,27 +10,31 @@
 use ResellerClub\Api;
 use ResellerClub\Config;
 use ResellerClub\Dns\Cname\CnameRecord;
+use ResellerClub\Dns\Cname\Requests\AddRequest;
 use ResellerClub\Dns\Cname\Requests\UpdateRequest;
+use ResellerClub\Dns\Cname\Responses\AddResponse;
 use ResellerClub\Dns\Cname\Responses\UpdateResponse;
 use ResellerClub\TimeToLive;
 
 class CnameRecordTest extends TestCase
 {
-    public function testUpdateInstance()
+    public function testAddInstance()
     {
-        $mock = new MockHandler([
-            new Response(
-                200,
-                ['Content-Type' => 'application/json'],
-                json_encode([
-                    'response' => [
-                        'status' => 'Success',
-                    ],
-                ])
-            ),
-        ]);
+        $cnameRecord = new CnameRecord($this->api($this->mock()));
+
+        $addRequest = new AddRequest(
+            'mytestdomain.com',
+            'www',
+            'cname.new.com',
+            new TimeToLive(7200)
+        );
 
-        $cnameRecord = new CnameRecord($this->api($mock));
+        $this->assertInstanceOf(AddResponse::class, $cnameRecord->add($addRequest));
+    }
+
+    public function testUpdateInstance()
+    {
+        $cnameRecord = new CnameRecord($this->api($this->mock()));
```

Comment:
If you're doing this more than once you could put it into setup i.e.
```
protected function setUp()
{
     $this->cnameRecord = new CnameRecord($this->api($this->mockResponse()));
}
```

Reply:
Agree, but not now with it affecting tests only